x/gov/migrations/v5/store_test.go

```diff
@@ -29,23 +27,20 @@ func TestMigrateStore(t *testing.T) {
 	constitutionCollection := collections.NewItem(sb, v5.ConstitutionKey, "constitution", collections.StringValue)
 
 	var params v1.Params
-	bz := store.Get(v4.ParamsKey)
+	bz := store.Get(v5.ParamsKey)
 	require.NoError(t, cdc.Unmarshal(bz, &params))
 	require.NotNil(t, params)
-	require.Equal(t, "", params.ExpeditedThreshold)
-	require.Equal(t, (*time.Duration)(nil), params.ExpeditedVotingPeriod)
+
+	// TODO: verify store migration for newly added params
```

Comment:
In case this TODO refers to this: yes there is also a v5 migration in atomone/x/gov, we should probably merge them.
https://github.com/atomone-hub/atomone/blob/main/x/gov/migrations/v5/store.go